drivers/bridge/bridge.go

```diff
@@ -144,7 +144,7 @@ func newDriver() *driver {
 }
 
 // Init registers a new instance of bridge driver
-func Init(dc driverapi.DriverCallback, config map[string]interface{}) error {
+func Init(dc driverapi.DriverCallback, ps interface{}, config map[string]interface{}) error {
```

Comment:
plugingetter is only used in remote driver code. So it can be abstracted from other code paths. i.e bridge code doesnt need to know about a plugingetter, since it doesnt use it.